swig/python/flamegpu.i

```diff
@@ -654,6 +656,19 @@ class ModelVis;
 // Don't flatnest this, range is explicitly not included incase of GC related issues.
 %include "flamegpu/util/nvtx.h"
 
+// Include the detail'd telemetry test reporting, this is not intended to be user facing but must be wrapped so it can be called by pytest.
+// Due to swig's ignoring / renaming, we have to uningnore detail, rename, then reignore detail (I think)
+
+// unignore detail
+%rename("%s") flamegpu::detail;
+// Rename test suite telemetry detail/private api function, prior to ignoring detail. 
+// Because of how _ prefixes are handeled, this must be called via pyflamegpu._pyflamegpu.__TestSuiteTelemetry_sendResults()
+%rename (__TestSuiteTelemetry) flamegpu::detail::TestSuiteTelemetry;
+%include "flamegpu/detail/TestSuiteTelemetry.h"
+// Ignore detail agian? 
```

Comment:
Just whatever is ineffect at the time the file is included afaik

Reply:
Yes, that's my now understanding too. the ? isn't strictly required anymore (neither is re-ignoring, but it will prevent future detail wrapping if included after that line. 

This was the only combination of rename/include/ingnore statements that produced teh desired wrapping behaviour. if the unignoring didnt' work, the wrapped version of `TestSuiteTelemetry` wouldn't exist.